### DISCUSSION TITLE
High Availability (HA) for kubernetes-replicator

### DISCUSSION THREAD

**Discussion by nitinatgh** [mittwald/kubernetes-replicator#128] [Q&A]
Hi,

I'd like to find out whether or not kubernetes-replicator can be scaled to more than 1 replica for HA without issues (i.e. conflicts etc..)?

I've been trying to look for docs based on this, but currently I have not been successful in doing so. Any help or guidance around kubernetes-replicator for HA would be greatly appreciated. Other controllers I have seen make use of the Leader Election feature, is there any plans for this?

Also for the replica values in [deployment.yaml](https://github.com/mittwald/kubernetes-replicator/blob/master/deploy/helm-chart/kubernetes-replicator/templates/deployment.yaml#L8), it's set to 1, doesn't look like this can be overridden via the values.yaml, making me believe it cannot be scaled?

Thanks, look forward to your reply

Nitin

**Comment by schantier** [comment]
Hi,

Do we have news, on the possibility of having the application in HA.

because the subject because we had a replication problem following the crash of the pod

**Comment by alexdyas** [comment]
I've been experimenting with running kubernetes-replicator as a daemonset and so far I've not seen any issues.
